MarvelCharsApp/MarvelCharsApp/Helpers/TextHelper.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  TextHelper.swift
+//  MarvelCharsApp
+//
+//  Created by User on 3/12/21.
+//
+
+import Foundation
+import SwiftUI
+
+extension Text {
+    func abilitiesStyles() -> some View {
+        self.foregroundColor(Color(UIColor.primary_white))
+            .frame(width: 110, alignment: .leading)
+            .font(.custom("gilroy-regular", size: 16))
```

Comment:
set this font as a global constant variable, also find all ocurrences of this word on the code files and change them